mapmatcher/src/main/java/io/github/nik9000/mapmatcher/ListMatcher.java

```diff
@@ -41,7 +41,19 @@ public static ListMatcher matchesList() {
   public static ListMatcher matchesList(List<?> list) {
     ListMatcher matcher = matchesList();
     for (Object item : list) {
-      matcher = matcher.item(item);
+      matcher = matcher.item(item, false);
+    }
+    return matcher;
+  }
+
+  /*
+  * Create a {@linkplain ListMatcher} that matches a list
+  * with extraOk param which will applied to the children
```

Comment:
Maybe `@param extraOkForChildrenMap if true maps contained in the list allow extra fields`